project-example/spatialos.json

```diff
@@ -0,0 +1,8 @@
+{
+  "configurationVersion": "0.1",
+  "projectName": "xxx",
+  "schemaDescriptor": "./build/schema/schema.descriptor",
+  "serverWorkers": [
```

Comment:
Please add the client worker back in as seen here - https://github.com/jamiebrynes7/spatialos-sdk-rs/blob/master/spatialos-sdk/examples/project-example/spatialos/spatialos.json

Reply:
Ah right, I had deleted that to see if it would even work without a client worker JSON specified and I forgot to put it back. Sorry about that!

Is there any reason to have the client worker JSON? There isn't a client worker in the example project as far as I can tell.

Reply:
You can run a "client" worker by passing in the `--worker-type RustClient` flag to the project. I was planning on expanding this such that the behaviour did actually differ between the workers